back/sssscs/src/main/java/com/ib/user/User.java

```diff
@@ -19,12 +25,48 @@
 @AllArgsConstructor
 @Entity
 @Table(name = "users") // Must not be "user".
-public class User {
-	@Id 
+public class User implements UserDetails {
+
+	public enum Role {
+		FOO,
+	}
+
+	@Id
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
 	private Long id;
 	private String email;
```

Comment:
I think that the phone number is one of the required `User` fields as well.

Reply:
https://github.com/magley/sssscs/pull/2/commits/9211a0d36efcb18edf5741710eb7bc8e759fce5f

Reply:
Aside from the missing extra validation rules for the phone number, there should generally be a stronger validation of the registration form on the frontend.
For example, the example JSON body for the postman of the endpoint is only 4 fields long, but it passes the registration. Everything should _at least_ be required.

Reply:
Well actually I did mean validation on the _frontent_, having each text field marked as required, but no harm in doing backend validation now, I guess.